src/backends/s3/objects.ts

```diff
@@ -55,7 +55,6 @@ export async function getObject(
   if (successResponse.status !== 200 && successResponse.status !== 206) {
     const errMessage = `Get Object Failed: ${successResponse.statusText}`;
     logger.warn(errMessage);
```

Comment:
info might be better? or even differentiate the 404 as normal, with other codes may indeed need a warning and report to sentry?

Reply:
warning should depend on status code, a 404 is expected but not the other where the sentry report/warning should remain